### ISSUE TITLE
Add support PropertiesToExclude to BatchUpdateAsync

### ISSUE THREAD

**Issue by UvarovFedor** [borisdj/EFCore.BulkExtensions#603]
It is possible add support PropertiesToExclude to BatchUpdateAsync or check enum default value
I have model:
```
public class Source{
public Status StatusId { get; set; }
public Type TypeId{get;set;}
}
public enum Status : byte{ New, Old } 
public enum Type : byte { Undefined, Type1, Type2}
```
and update value as this:
```
var updateValues = new Source(){StatusId = fixState};
var updateColumns = new List<string>() { nameof(updateValues.StatusId) };
var updated = await context.Sources.Where(e => e.StatusId == faultState).BatchUpdateAsync(updateValues, updateColumns);
```
when BatchUpdateAsync search default value for TypeId column - it find Type.Undefined
```
object propertyUpdateValue = property.GetValue(updateValues); // Type.Undefined
object propertyDefaultValue = property.GetValue(defaultValues); // Type.Undefined
```
but when check ConvertibleColumnConverterDict
```
if (tableInfo.ConvertibleColumnConverterDict.ContainsKey(columnName))
{
      propertyUpdateValue = tableInfo.ConvertibleColumnConverterDict[columnName].ConvertToProvider.Invoke(propertyUpdateValue);
     // now propertyUpdateValue == (byte)0
}
```
it replace Type.Undefined  with (byte)0. and value become changed!
```
bool isDifferentFromDefault = propertyUpdateValue != null && propertyUpdateValue?.ToString() != propertyDefaultValue?.ToString();
// isDifferentFromDefault == true
```

**Comment by borisdj** [comment]
I think it should work using Expression arg.:
`context.Sources.Where(e => e.StatusId == faultState).BatchUpdate(a => new Source { StatusId = fixState });`
Will see to fix via simple object also.

**Comment by UvarovFedor** [comment]
it works, thanks!

**Comment by borisdj** [comment]
Fixed with v5.3.2 for simple object as well.
